oddbird/oddleventy
# [event-wpblocks] WordPress Naperville Block Talk post

## Cute animal pic
![giorgio-trovato-a7RM50DM8pk-unsplash](https://user-images.githubusercontent.com/1581694/108030466-e800c500-6ff4-11eb-8f46-e3beff075ddb.jpg)

## Link to Trello card (if applicable)
https://trello.com/c/6eXkNwPY/124-wp-blocks-demo-talk-wp-naperville

## Description
Add video blog post about the WP Naperville Meetup Demo in November

## Steps to test/reproduce
go to /talks/wordpress-block-enhancements/ to see talk single. Also, view the resources landing page to preview the thumbnail view.

## Show me
![Screen Shot 2021-02-16 at 1 24 22 AM](https://user-images.githubusercontent.com/1581694/108031135-ebe11700-6ff5-11eb-8c64-4f327196a2d7.jpg)

![screencapture-localhost-8080-talks-wordpress-block-enhancements-2021-02-16-01_24_59](https://user-images.githubusercontent.com/1581694/108031304-3e223800-6ff6-11eb-8796-064ce0b46bed.png)
